metrics_sdk/test/opentelemetry/sdk/metrics/instrument/counter_test.rb

```diff
@@ -30,4 +30,30 @@
     _(last_snapshot[0].data_points[0].attributes).must_equal('foo' => 'bar')
     _(last_snapshot[0].aggregation_temporality).must_equal(:delta)
   end
+
+  describe 'with advisory parameters' do
+    let(:random_key) { "a#{SecureRandom.hex}" }
```

Comment:
The tests on 3.0 are failing with a missing SecureRandom require: https://github.com/open-telemetry/opentelemetry-ruby/actions/runs/13058885253/job/36668636949?pr=1801#step:6:185

Reply:
Also, similar question to the API -- what do you think about adding advisory parameters/attributes for the other instruments?